src/index.js

```diff
@@ -28,45 +33,86 @@ const arr = [
   },
 ];
 
-for (let i = 0; i < arr.length; i += 1) {
-  const dropZone = document.createElement('div');
-  const inputlabel = document.createElement('label');
-  const inputCheck = document.createElement('input');
-  const inputsLabelsContainer = document.createElement('div');
-
-  dropZone.className = 'dropZone';
-  clearAll.className = 'clearAll';
-  inputsLabelsContainer.className = 'inputsLabelsContainer';
-
-  dropZone.id = `draggable-${i}`;
-  inputCheck.id = `inputCheck-${i}`;
-
-  dropZone.setAttribute('draggable', 'true');
-  inputlabel.setAttribute('for', `inputCheck-${i}`);
-  inputCheck.setAttribute('type', 'checkbox');
+if (window.localStorage.length == null || window.localStorage.length === 0) {
```

Comment:
- Very smart to check if the local storage is cleared before rendering the list elements, :+1: . But you should mention it in your readme or PR summary as it may not be noticed easily by others and it can be mistakenly related to some other issues that the lists are not rendering. Anyways, you did a great job.